mappings/net/minecraft/world/chunk/WrappedChunk.mapping

```diff
@@ -1,8 +1,10 @@
-CLASS net/minecraft/class_2821 net/minecraft/world/chunk/ReadOnlyChunk
+CLASS net/minecraft/class_2821 net/minecraft/world/chunk/WrappedChunk
 	COMMENT Represents a read only view of a world chunk used in world generation.
```

Comment:
We need to update this javadoc as well.

Reply:
Probably going to move this change out to a separate PR and change the name to either `WrapperChunk` or `WrapperProtoChunk` - will make sure to update the JavaDoc there